src/index.html

```diff
@@ -6,8 +6,28 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__image"></div>
+      <a class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+      <span class="card__code">Product code: 195434</span>
+      <div class="card__reviews reviews">
```

Comment:
Always add space between same blocks in same parent if they are not the same and don`t take only one line like in stars block